api/src/views/script-runner.hbs

```diff
@@ -0,0 +1,18 @@
+<h1>{{t "t.hello"}} {{> user-name }}</h1>
+<p>
+  {{t "scriptRunner.information"}}
+</p>
+
+<p>
+  {{t "scriptRunner.yourApplicationNumber" application}}
```

Comment:
Does this need to be something more specific than `application`?

Reply:
Hello! its a bit hard to test in here but I tested with my script runner pr and the email ends up looking like: 
![image](https://github.com/bloom-housing/bloom/assets/10088764/4d53c732-8d53-4059-9a9c-b81c29f7313e)

Reply:
if that looks good I think we can merge and I'll put out the pr that lets us test this more easily!

Reply:
yep yep!